contracts/common/MultiChain.sol

```diff
@@ -0,0 +1,48 @@
+/*
+  This file is part of The Colony Network.
+
+  The Colony Network is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  The Colony Network is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with The Colony Network. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+pragma solidity 0.7.3;
+
+
+
+contract MultiChain {
+  function chainId() internal view returns (uint256) {
```

Comment:
Nitpick: can we call this function `getChainId`? Also no reason not to make it public.

Reply:
It's weird to make this public without exposing it on the resolver though - it's only being made public for the tests, and I have a well documented hatred of changing production code for the benefit of the tests :stuck_out_tongue:

Reply:
Okay I'll make it external and it shouldn't cost more gas as far as I can tell.

Reply:
Yeah, I hear you about the resolver. Happy leaving it internal I suppose...